packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -141,6 +160,13 @@ export default class QuanticGeneratedAnswer extends LightningElement {
 
   renderedCallback() {
     initializeWithHeadless(this, this.engineId, this.initialize);
+    if (this.collapsible) {
+      const answerElementHeight = this.isStreaming
+        ? this.generatedAnswerElementHeight + 50
```

Comment:
I thought I added a comment in the code to explain that, I'll re-add it.

In short it's because we're still streaming so we "know" the answer isn't yet complete. To avoid some jankyness, I pre-suppose the answer will at least be +50 px in height in order to start displaying the fade-out effect sooner.

This +50 disappears once the answer is no longer streaming so we can take the real height at this point.